example/simple/main.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
```

Comment:
The copyright header and the package comment should be separate comments. Like this:

```Go
// Copyright 2017 The go-github AUTHORS. All rights reserved.
//
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// The simple command demonstrates a simple functionality which
// prompts the user for a GitHub username and lists all the public
// organization memberships of the specified username.
package main
```

Notice the blank line separating the comments.

Otherwise the entire comment is treated as the package comment; that's not what we want to see in godoc.

Reply:
Got it. Thanks for the explanation.